hip-clang.docker

```diff
@@ -6,8 +6,7 @@ ARG PREFIX=/usr/local
 RUN dpkg --add-architecture i386
 
 # Add rocm repository
-RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://compute-artifactory.amd.com/artifactory/list/rocm-osdb-deb/ compute-rocm-dkms-no-npi-hipclang 1927 > /etc/apt/sources.list.d/rocm.list'
-
+RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://compute-artifactory.amd.com/artifactory/list/rocm-osdb-deb/ compute-rocm-dkms-no-npi-hipclang 2272 > /etc/apt/sources.list.d/rocm.list'
```

Comment:
+1 as currently I see some strange failures with `comgr-attach` branch and suspect that comgr is not fully functional in 1927.